src/Hng.Application/Features/Timezones/Commands/CreateTimezoneCommand.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Hng.Application.Features.Timezones.Dtos;
+using MediatR;
+
+namespace Hng.Application.Features.Timezones.Commands
+{
+    public class CreateTimezoneCommand : IRequest<CreateTimezoneResponseDto>
+    {
+        public string Timezone { get; set; }
+        public string GmtOffset { get; set; }
+        public string Description { get; set; }
```

Comment:
Include JsonPropertyName on all dtos for snake_case

Reply:
Done